api-report/client-utils.api.md

```diff
@@ -17,11 +17,11 @@ export class Buffer extends Uint8Array {
     // (undocumented)
     static isBuffer(obj: unknown): obj is Buffer;
     // (undocumented)
-    toString(encoding?: string): string;
+    toString(encoding?: "utf8" | "utf-8" | "base64"): string;
```

Comment:
TS5's improved type checking notices that the 'encoding' string is passed to Buffer.toString(), which only accepts specific values.

I've narrowed the type of 'encoding' to just the values that IsoBuffer supports (i.e., those supported in both Node.js and Browser environments.)